.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .idea
 .DS_STORE
+*.pyc
+main.db
```

Comment:
Thanks! It can be on README.md, actually. :-) Our current README here is so short ☹️